### PR TITLE
Check pre-commit hooks are passing

### PR DESCRIPTION
**Pull request by arnaudgelas**
## What does this PR do?

Fixes # (issue) <- this [links related issue to this PR](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: .github/workflows/code-formatting.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
     branches: [master, "release/*"]
 
 jobs:
-  imports-check-isort:
-    name: Check valid import formatting with isort
+  pre-commit-check:
```
Comment (Borda) [inline .github/workflows/code-formatting.yml:10]: Let's separate pre-commit and isort...
if you think it is needed to add another job, but do not drop isort

**Review by Borda — CHANGES_REQUESTED** [review]

File: .github/workflows/code-formatting.yml (hunk 1 above)
Comment (carmocca) [reply .github/workflows/code-formatting.yml:10]: why? it makes sense to me to directly run pre-commit

and pre-commit outputs what failed/passed per hook

**Comment by Borda** [comment]
@arnaudgelas how is it going here? :] I do not think we want to enforce anyone to use pre-commit...

**Comment by carmocca** [comment]
HI @arnaudgelas!

After chatting with @Borda, we chose to rely on pre-commit but do it on separate jobs so it is easier to see which hooks failed.

**Comment by Borda** [comment]
> Can you update this PR to do it?

I have fixed it now :]

**Review by Borda — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]
⏫  based on Borda accept :)

**Comment by carmocca** [comment]
formatting checks have been removed so this no longer applies. Closing.

**Pull request closed without merging**